test/contrib/test_tfp.py

```diff
@@ -102,3 +103,62 @@ def model(data):
     mcmc.print_summary()
     samples = mcmc.get_samples()
     assert_allclose(jnp.mean(samples['p_latent'], 0), true_probs, atol=0.05)
+
+
+@pytest.mark.parametrize('kernel, kwargs', [
+    ('RandomWalkMetropolis', dict()),
+    ('SliceSampler', dict(step_size=1.0, max_doublings=5))
```

Comment:
How many other samplers are there, and we tested if the wrappers work as expected?

Reply:
There are several more. Each of them has different `kwargs`. I'll loop over them and add corresponding kwargs in the test.

Reply:
@neerajprad I have added tests for remaining kernels. Only [ReplicaExchangeMC](https://www.tensorflow.org/probability/api_docs/python/tfp/substrates/jax/mcmc/ReplicaExchangeMC) is not working due to some of its reshape logics. I think we need to do something like `vmap(potential_fn)` so the potential_fn can be evaluated on multiple replicas of params. I will try to support it in a follow-up PR if possible...

Reply:
Just want to update, ReplicaExchangeMC should work now. I think we have supported all TFP kernels. And any user-defined tfp kernel can be converted to a NumPyro compatible kernel with `TFPKernel[custom_kernel]`. Cheers, :D